client/components/dangerousHtml/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+export default React.createClass( {
+	propTypes: {
+		tag: React.PropTypes.string.isRequired,
```

Comment:
That's a good idea. I actually kind of thought of that when I was writing the README.